CITATION.cff

```diff
@@ -0,0 +1,68 @@
+cff-version: 1.2.0
+abstract: "The Core Imaging Library (CIL) is an open-source Python framework for tomographic imaging 
+           with particular emphasis on reconstruction of challenging datasets. Conventional filtered 
+           backprojection reconstruction tends to be insufficient for highly noisy, incomplete, non-standard
+           or multichannel data arising for example in dynamic, spectral and in situ tomography. CIL 
+           provides an extensive modular optimization framework for prototyping reconstruction methods 
+           including sparsity and total variation regularization, as well as tools for loading, preprocessing 
+           and visualizing tomographic data."
+authors:
+  - name: UK Research and Innovation and University of Manchester
+date-released: "08-01-18"
+identifiers:
+  - description: "This is the collection of archived snapshots of all versions of the Core Imaging Library"
+    type: doi
+    value: 10.5281/zenodo.4746198
+  - description: "This is the archived snapshot of version 21.3.1 of the Core Imaging Library"
+    type: doi
+    value: 10.5281/zenodo.5825832
+  - description: "This is the archived snapshot of version 21.3.0 of the Core Imaging Library"
+    type: doi
+    value: 10.5281/zenodo.5752905
+  - description: "This is the archived snapshot of version 21.2.0 of the Core Imaging Library"
+    type: doi
+    value: 10.5281/zenodo.5012886
+keywords:
+  - "tomographic imaging"
+  - research
+  - tomography
+  - reconstruction
+  - imaging
+  - hyperspectral
+  - optimisation
+license: Apache-2.0
+message: "If you use this software, please cite both the article from preferred-citation and the software itself."
+preferred-citation:
+  authors:
+    - family-names: Jørgensen
+      given-names: Jakob Sauer
+    - family-names: Ametova
+      given-names: Evelina
+    - family-names: Burca
+      given-names: Genoveva
+    - family-names: Fardell
+      given-names: Gemma
+    - family-names: Papoutsellis
+      given-names: Evangelos
+    - family-names: Pasca
+      given-names: Edoardo
+    - family-names: Thielemans
+      given-names: Kris
+    - family-names: Turner
+      given-names: Martin
+    - family-names: Warr
+      given-names: Ryan
+    - family-names: Lionheart
+      given-names: William RB
+    - family-names: Withers
+      given-names: Philip J     
+  title: "Core Imaging Library - Part I: a versatile Python framework for tomographic imaging"
+  type: article
+  doi: 10.1098/rsta.2020.0192
+  issue: 2204
+  volume: 379
+  journal: "Phylosophical Transaction of The Royal Society A"
```

Comment:
Should be Ph`i`losophical Transaction`s`

Reply:
I can't even copy!